src/Symfony/Bundle/TwigBundle/RuntimeExtensionInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle;
+
+/**
+ * Provide a marker to automatically add the "twig.runtime" tag to the
+ * associated service.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+interface RuntimeExtensionInterface
```

Comment:
But it is useless to tag the runtime interface on their side. It is only useful in a symfony context. 

What is the benefits of moving it to twig ?